### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by movence**
*Description of changes:*
* Upgrade CWAgent to v1.300044.0
* Upgrade CWAgent Operator to v1.6.0
* Release notes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by musa-asad — DISMISSED** [review]

**Comment by movence** [comment]
The failed windows integ tests are expected as the agent ECR image not available in the public repo. The windows terraform checks `fluentbit` and `agent` [daemonset status explicitly](https://github.com/aws-observability/helm-charts/blob/main/integration-tests/amazon-cloudwatch-observability/terraform/helm-windows/main.tf#L225) that makes the test to fail.

File: RELEASE_NOTES [hunk 1]
```diff
@@ -1,3 +1,14 @@
+=======================================================================
+amazon-cloudwatch-observability v2.0.0 (2024-08-15)
+========================================================================
+Enhancements:
+* Allow configurable resource requests and limits through command arguments for AppSignals (#65)
```
Comment (lisguo) [inline RELEASE_NOTES:5]: I think we want to mention that we have a breaking change section to say we have enforced default requests and limits for auto instrumentation init containers. Similar to the opentelemetry-collector releases: https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.107.0


```
Breaking Changes:
* Enforce default requests and limits for auto instrumentation init containers
Enhancements:
* Allow configurable requests and limits for auto instrumentation init containers
```

Comment (mitali-salvi) [reply RELEASE_NOTES:5]: +1 to that. additionally lets capture that as part of the helm-chart release as well

Comment (movence) [reply RELEASE_NOTES:5]: yes, I was going to adding those breaking changes to Github release page while keeping the release notes format as-is, but I agree that they are better to be in sync.

**Review by mitali-salvi — APPROVED** [review]

**Review by lisguo — APPROVED** [review]

**Pull request merged**